Automattic/pocket-casts-android
# Import and export tracks

## Description
Tracks for the import and export screen in settings.

## Testing Instructions
1. Open the app, go to the Profile tab → ⚙️ → `Import & export OPML`
2. ✅ Observe the event `settings_import_shown`
3. Tap on "Select file"
4. ✅ Observe the event `settings_import_select_file`
5. Import an OPML file
6. ✅ Observe the event `opml_import_started`
7. ✅ Observe the event `opml_import_finished, Properties: {"number":[number of podcasts in the opml], ... }`
8. Tap on "By URL"
9. ✅ Observe the event `settings_import_by_url`
10. Enter a URL pointing at an opml file with podcasts ([for example](https://gist.githubusercontent.com/mchowning/905f27f0ab3c863cbc39d75f92910970/raw/be374a9a2b05eb5ea28b0d5bbe67e4b0e86f6de7/opml_example))
11. ✅ Observe the events: 
    1. `opml_import_started`
    2. `opml_import_finished, Properties: {"number":[number of podcasts in the opml], ... }`
14. Tap on "Send email"
15. ✅ Observe the events:
    1. `settings_import_export_email_tapped`
    2. `settings_import_export_started`
    3. `settings_import_export_finished`
16. Close the share sheet
17. Turn on airplane mode
18. Tap on "Send email" again
19. ✅ Observe the events:
    1. `settings_import_export_email_tapped`
    2. `settings_import_export_started`
    3. `settings_import_export_failed, Properties: {"reason":"server_call_failure", ... }`
20. Close the share sheet
21. Tap on "Save file"
22. ✅ Observe the events:
    1. `settings_import_export_file_tapped`
    2. `settings_import_export_started`
    3. `settings_import_export_finished`
23. Close the share sheet
24. Turn on airplane mode
25. Tap on "Save file" again
26. ✅ Observe the events:
    1. `settings_import_export_file_tapped`
    2. `settings_import_export_started`
    3. `settings_import_export_failed, Properties: {"reason":"server_call_failure", ... }`

## Checklist
